src/EA.Weee.Requests/AatfEvidence/GetEvidenceNotesByOrganisationRequest.cs

```diff
@@ -8,9 +8,13 @@
 
     public class GetEvidenceNotesByOrganisationRequest : EvidenceNoteFilterBaseRequest
     {
+        public bool TransferredOut { get; private set; }
+
         public GetEvidenceNotesByOrganisationRequest(Guid organisationId, 
             List<NoteStatus> allowedStatuses,
-            int complianceYear, NoteType noteTypeFilter)
+            int complianceYear, 
+            NoteType noteTypeFilter,
```

Comment:
does it need to be a list? in case you want to see both types on the same tab

Reply:
yeah will make that update when we know how the tabs should be working for transfer notes